vendor/assets/stylesheets/dvl/core/grid.scss

```diff
@@ -11,6 +11,9 @@
   padding: 0;
   list-style: none; /* 2 */
   font-size: 0; /* 3 */
+  @media screen and (min-width: $lapWidth) {
```

Comment:
Nope, units are relative to font size, so they don't change when the viewport changes.

It's pretty simple. Larger screen sizes = larger gutter width.